src/wallet/wallet2.cpp

```diff
@@ -5808,27 +5811,6 @@ void wallet2::restore(const std::string& wallet_, const epee::wipeable_string& p
   }
 }
 //----------------------------------------------------------------------------------------------------
-epee::misc_utils::auto_scope_leave_caller wallet2::decrypt_account_for_multisig(const epee::wipeable_string &password)
```

Comment:
So this deleted chunk here was basically just duplicated code?

Reply:
Yeah, basically this code does the same thing as `wallet_keys_unlocker`